python/ray/serve/scripts.py

```diff
@@ -128,6 +128,15 @@ def convert_args_to_dict(args: Tuple[str]) -> Dict[str, str]:
     return args_dict
 
 
+def warn_if_agent_address_set():
+    if "RAY_AGENT_ADDRESS" in os.environ:
+        cli_logger.warning(
+            "The `RAY_AGENT_ADDRESS` env var has been deprecated in favor of "
+            "the `RAY_HEAD_ADDRESS` env var. The `RAY_AGENT_ADDRESS` is "
```

Comment:
I think you mean `RAY_DASHBOARD_ADDRESS`?

Reply:
Yeah, thanks for the catch!